uttut/pipeline/edit/base.py

```diff
@@ -4,50 +4,50 @@
 from .validation import _validate_start_end
 
 
-class Edit(ABC):
+class Replacement(ABC):
     '''Represents an edit to a Sequence such as a string or a list
 
     A sequence is a collection of objects that can be indexed in a linear fashion. E.g. a string
 
-    A edit to a sequence is a replacement of a continuous range of the sequence with the elements
-    of the replacement sequence.
+    A replacement to a sequence is a new_value of a continuous range of the sequence with
+    the elements of the new_value sequence.
```

Comment:
A replacement maps a slice of original sequence to a new value.

Reply:
@absolutelyNoWarranty

Reply:
A replacement is a data structure that records the procedure of replacing a slice of original sequence with 
new_value.